common/src/main/java/com/wynntils/models/activities/caves/CaveModel.java

```diff
@@ -70,24 +70,27 @@ private List<CaveInfo> sortCaveInfoList(ActivitySortOrder sortOrder, List<CaveIn
         Comparator<CaveInfo> baseComparator =
                 Comparator.comparing(caveInfo -> !caveInfo.name().equals(trackedCaveName));
         return switch (sortOrder) {
-            case LEVEL -> caveList.stream()
-                    .sorted(baseComparator
-                            .thenComparing(CaveInfo::status)
-                            .thenComparing(CaveInfo::recommendedLevel)
-                            .thenComparing(CaveInfo::name))
-                    .toList();
-            case DISTANCE -> caveList.stream()
-                    .sorted(baseComparator
-                            .thenComparing(CaveInfo::status)
-                            .thenComparing(CaveInfo::distance)
-                            .thenComparing(CaveInfo::name))
-                    .toList();
-            case ALPHABETIC -> caveList.stream()
-                    .sorted(baseComparator
-                            .thenComparing(CaveInfo::status)
-                            .thenComparing(CaveInfo::name)
-                            .thenComparing(CaveInfo::recommendedLevel))
-                    .toList();
+            case LEVEL ->
```

Comment:
What happened here? Spotless acting up?

Reply:
New Palantir format I think.

Reply:
Ok. *sigh* I don't like it, it just became more spacious for no good reason. But I've tied myself to the mast so I guess I'll just have to shut up and like whatever Spotless gives me...